### PR TITLE
Fix: Show legacy web headers

### LINKED ISSUE #497: Legacy web headers do not display in the UI and cannot be deleted via API
**Describe the bug**
Instances with legacy web headers are seeing head tags rendering but have no way of editing/deleting them.

**To Reproduce**
1. Please see ticket for instance reference: https://zestysupport.zendesk.com/agent/tickets/1208
2. See that head tag in question is not visible in the instance Settings > Headtags section
3. The only way to pull this legacy headtag is using this endpoint: https://instances-api.zesty.org/#1b097314-1e85-450f-86f7-186f1a3f080f 
4. There is no endpoint to delete/edit the headtag

**Expected behavior**
Expect to be able to see and edit all head tags which are being loaded by Zesty in the Manager UI

**Additional context**
This has been brought up by a client, please reference the ZenDesk ticket linked above or message me directly :)

### PR DESCRIPTION
**Pull request by theofficialnar**
Closes #497 

#### Preview
Screenshot from 2023-03-07 13-56-10

### CODE REVIEW

File: src/shell/services/headTags.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
+import instanceZUID from "../../utility/instanceZUID";
+import { getResponseData, prepareHeaders } from "./util";
+import { LegacyHeader } from "./types";
+
+export const headTagApi = createApi({
```
[the remark this answers is not in the document]
Comment (theofficialnar) [reply src/shell/services/headTags.ts]: Gotcha, moved to instance.ts

File: src/shell/services/headTags.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    // @ts-ignore
+    baseUrl: `${__CONFIG__.API_INSTANCE_PROTOCOL}${instanceZUID}${__CONFIG__.API_INSTANCE}`,
+    prepareHeaders,
+  }),
+  tagTypes: ["LegacyHeadTags"],
+  endpoints: (builder) => ({
+    getLegacyHeadTags: builder.query<LegacyHeader[], void>({
+      query: () => `/web/headers`,
+      transformResponse: getResponseData,
+      keepUnusedDataFor: 0.0001,
```
[the remark this answers is not in the document]
Comment (theofficialnar) [reply src/shell/services/headTags.ts]: Forgot to remove this when I copied the the block off of an already existing query

File: src/shell/services/headTags.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      providesTags: () => [{ type: "LegacyHeadTags" }],
```
[the remark this answers is not in the document]
Comment (theofficialnar) [reply src/shell/services/headTags.ts]: Removed on latest update

**Comment by giseleblair** [comment]
@theofficialnar besides the warning that there are legacy headtags present, do we display which headtags are legacy?

**Comment by theofficialnar** [comment]
> @theofficialnar besides the warning that there are legacy headtags present, do we display which headtags are legacy?

@giseleblair yup, I've added it to the preview section of the head tag. Only issue I see is that, the users will probably need to scan through the input fields first to find out which of these are considered legacy since I haven't indicated it on the preview - I honestly thought we just had to also make sure it's added there.

**Pull request merged**